src/components-examples/material/tooltip/tooltip-harness/tooltip-harness-example.spec.ts

```diff
@@ -0,0 +1,86 @@
+import {TestBed, ComponentFixture, waitForAsync} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatTooltipHarness} from '@angular/material/tooltip/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {MatTooltipModule} from '@angular/material/tooltip';
+import {TooltipHarnessExample} from './tooltip-harness-example';
+import {Platform} from '@angular/cdk/platform';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+
+describe('TooltipHarnessExample', () => {
+  let fixture: ComponentFixture<TooltipHarnessExample>;
+  let loader: HarnessLoader;
+  let isSupported: boolean;
+
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatTooltipModule, NoopAnimationsModule],
+        declarations: [TooltipHarnessExample]
+      }).compileComponents();
+      fixture = TestBed.createComponent(TooltipHarnessExample);
+      fixture.detectChanges();
+      loader = TestbedHarnessEnvironment.loader(fixture);
+      const platform = TestBed.inject(Platform);
+      isSupported = !platform.IOS && !platform.ANDROID;
+    })
+  );
+
+  it('should load all tooltip harnesses', async () => {
+    if (!isSupported) {
```

Comment:
I think that we should remove these checks from the harness examples. We need it in our own tests, because we run them against iOS and Android device, but most people won't be doing it. It was also a limitation that we should be able to work around now.